1.7/administration/opt-out.md

```diff
@@ -5,7 +5,7 @@ menu_order: 6
 
 ## Authentication
 
-You can opt-out of the provided authentication by disabling it for your cluster. To disable authentication, add this parameter to your [`config.yaml`][4] file during installation (note this requires using the [CLI][1] or [advanced][2] installers):
+You can opt-out of the provided authentication by disabling it for your cluster. To disable authentication, add this parameter to your [`config.yaml`](/docs/1.7/administration/installing/custom/configuration-parameters/) file during installation (note this requires using the [CLI][1] or [advanced][2] installers):
```

Comment:
Please keep the links at the bottom. It makes it *way* easier to update the links.